src/Effects.js

```diff
@@ -600,6 +600,36 @@ module.exports = class Effects {
         });
       }
     };
+
+    this.bubbles = {
+      bubble: [],
+      draw: function () {
+        let bubble = {
+          x: p5.random(-100, 400),
+          y: 410,
+          velocityX: p5.random(-2, 2),
+          size: p5.random(6, 12, 18),
+          color: randomColor(100, 50, 0.25),
```

Comment:
I seem to remember there were some issues (memory, I think) with rendering things with transparencies - @joshlory or @islemaster do you have any memory of this?